Makefile

```diff
@@ -1,3 +1,4 @@
+COMMON_OVERLAYS += tkl-webcp
```

Comment:
Perhaps I'm missing something, but I don't understand why we're including `tkl-webcp`? As I've been so slow to do the code review and I know you're busy @qq7, no need for you to do anything (unless of course you have time and inclination). Either way I'll have a closer look before publishing...

Reply:
@JedMeister this is required since nodejs common tries to copy tklweb-cp. It is being removed in the conf script.

Coming back to it, it surely is not the best solution and it would be better to run a check for if tklweb-cp exists in the common nodejs conf, what do you think?

Reply:
Ah ok, thanks mate.

I've actually refactored things a bit (more) in common (see https://github.com/turnkeylinux/common/pull/220).

So now if you want to pull in tkl-webcp for node, you can use [`common/mk/turnkey/nodejs.m`](https://github.com/turnkeylinux/common/blob/master/mk/turnkey/nodejs.mk). If you just need node, then you can use [`common/mk/turnkey/nodejs.mk`](https://github.com/turnkeylinux/common/blob/master/mk/turnkey/nodejs.mk).